linera-service/src/cli/command.rs

```diff
@@ -330,6 +330,10 @@ pub enum ClientCommand {
         #[arg(long)]
         write_operation: Option<Amount>,
 
+        /// Set the price per byte obtained from runtime.
```

Comment:
```suggestion
        /// Set the price per byte read from runtime.
```

(Or "obtained" below, but I think they should be the same.)

Reply:
Ok.